### PR TITLE
Add static JSONS for module's releases

### PR DESCRIPTION
**Pull request by marinalimeira**
# Description
[[Notion] Changelogs in the Docs website](https://www.notion.so/gruntwork/Changelogs-in-the-Docs-website-760d09945c244a9994859c7475bc8b91)
[[Jira] PATCHER-95](https://gruntwork.atlassian.net/browse/PATCHER-95)

This PR contains the generated code from https://github.com/gruntwork-io/docs-sourcer/pull/121.

For each module in each Terraform repo, the JSON represents whenever the module had _some_ change in that release.

### CODE REVIEW

File: static/patcher-changelogs/terraform-aws-service-catalog/mgmt/openvpn-server.json [hunk 1]
```diff
@@ -0,0 +1,1655 @@
+{
```
Comment (marinalimeira) [inline static/patcher-changelogs/terraform-aws-service-catalog/mgmt/openvpn-server.json:1]: We can compare this with the changelog.md https://github.com/gruntwork-io/terraform-aws-service-catalog/blob/master/modules/mgmt/openvpn-server/CHANGELOG.md, and have the same result!

File: static/patcher-changelogs/terraform-aws-static-assets/_docs.json
Comment (eak12913) [inline static/patcher-changelogs/terraform-aws-static-assets/_docs.json:1]: Another candidate for the ignore file

File: static/patcher-changelogs/terraform-aws-service-catalog/tls-scripts/Dockerfile.json
Comment (eak12913) [inline static/patcher-changelogs/terraform-aws-service-catalog/tls-scripts/Dockerfile.json:1]: Candidate for the ignore file

File: static/patcher-changelogs/terraform-aws-service-catalog/tls-scripts/install.sh.json
Comment (eak12913) [inline static/patcher-changelogs/terraform-aws-service-catalog/tls-scripts/install.sh.json:1]: I asked about this in the docs-sourcer PR - but I think we should consider shell scripts in the ignore list right?

File: static/patcher-changelogs/terraform-aws-security/_deprecated.json
[the remark this answers is not in the document]
Comment (marinalimeira) [reply static/patcher-changelogs/terraform-aws-security/_deprecated.json:1]: Removed - 6d5ff3e68a5. There was a conditional missing.

**Review by eak12913 — APPROVED** [review]
Approving with the extra scripts as we anticipate that Patcher will be switching to using a dedicated API in the future and we will be able to either iterate or deprecate the changes from this PR.

**Pull request merged**